roles/mesos_consul/templates/mesos_consul.service.j2

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Mesos Consul
```

Comment:
Thinking if we should run this through the weave proxy, probably does not matter